src/next.ts

```
@@ -1,310 +1,236 @@
+        console.warn("Failed to generate metadata:", error);
+        return opts.fallback || {};
       }
     };
   }
 
-  public generateMetadata<Props>(
-    generateMetadataOptions: GenerateMetadataOptions,
-  ): NextGenerateMetadata<Props>;
-  public generateMetadata<Props>(
-    generateMetadataFn: (
+  public getRootMetadata<Props = {}>(
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)